client/app/components/CreateEventModal.jsx

```diff
@@ -0,0 +1,181 @@
+import React, { PropTypes } from 'react';
+import { Button, Modal, FormGroup, ControlLabel, FormControl, Alert } from 'react-bootstrap';
+import DateTimeField from 'react-bootstrap-datetimepicker';
+import moment from 'moment';
+import axios from 'axios';
+import _ from 'lodash';
+
+const { func, bool, array } = PropTypes;
+
+const DATE_FORMAT = 'DD/MM/YYYY HH:mm';
+
+export default class CreateEventModal extends React.Component {
+  static propTypes = {
+    openModal:       func,
+    closeModal:      func,
+    showModal:       bool,
+    conferenceRooms: array
+  };
+
+  constructor() {
+    super();
+
+    const initialDate = moment().format(DATE_FORMAT);
+
+    this.state = {
+      showErrorMessage: false,
```

Comment:
I would probably be better to have an `error` field and store the error message in it. Instead of checking `showErrorMessage` just check if `error` exists. If you do that it's important to set `error` to `null` after a successful submission or changing input values.